hack/coverage.sh

```diff
@@ -0,0 +1,11 @@
+
+set -e
+echo "" > coverage.txt
+
+for d in $(go list ./... | grep -v '/vendor/\|/k8s'); do
+    go test -race -coverprofile=profile.out -covermode=atomic $d
+    if [ -f profile.out ]; then
+        cat profile.out >> coverage.txt
+        rm profile.out
+    fi
+done
```

Comment:
Add new lines properly. Rest looks good to me :+1:

Reply:
Addressed @satyamz Thank you.